app/controller/user/index.ts

```diff
@@ -9,45 +9,45 @@ const logger = setupLogger("user")
 
 let api = Router();
 
-api.post("/", async (req: Request, res: Response, next: NextFunction): Promise<any> => {
-    const user = new User(req.body);
-    try {
+api.post(
+    "/",
+    async (req: Request, res: Response, next: NextFunction): Promise<any> => {
+      const user = new User(req.body);
+      try {
         User.register(new User(user), req.body.password, (err, createdUser) => {
```

Comment:
You dont really need try catch when you are doing callbacks.
Use async await for getting full value of try catch